src/test/java/ua/biz/test/tests/APItest/workExperience/CreateEmployeeWorkExperienceTest.java

```diff
@@ -0,0 +1,13 @@
+package ua.biz.test.tests.APItest.workExperience;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ua.biz.test.base.BaseAPItest;
+
+public class CreateEmployeeWorkExperienceTest extends BaseAPItest {
+
+    @Test
+    public void testCreateEmployeeWorkExperience() {
+        Assert.assertEquals(createEmployeeWorkExperience(workExperience).getStatusCode(), 200);
+    }
```

Comment:
Verifing only status code is not enought for complete test verification. Needs to verify body as well. This remark related to all API tests.